estate_account/models/estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_state_sold(self):
+        invoice_vals_list = []
+        for record in self:
+            invoice_vals = {
+                'partner_id': record.buyer_id.id,
+                'move_type': 'out_invoice',
+                'journal_id': self.env['account.journal'].search([('type', '=', 'sale')], limit=1).id,
+                'invoice_line_ids': [
+                    Command.create({
+                        'name': "6% Commission",
+                        'quantity': 1,
+                        'price_unit': record.selling_price * 0.06,
+                    }),
+                    Command.create({
+                        'name': "Administrative fees",
+                        'quantity': 1,
+                        'price_unit': 100.0,
+                    }),
+                ],
+                }
+            invoice_vals_list.append(invoice_vals)
+        self.env['account.move'].create(invoice_vals_list)
+        return super().action_state_sold()
```

Comment:
It would be worth to call the super earlier, even more here in this case that you raise in some cases...